RecoMuon/L3TrackFinder/interface/HLTMuonL2SelectorForL3IO.h

```diff
@@ -0,0 +1,34 @@
+#ifndef RecoMuon_HLTMuonL2SelectorForL3IO_HLTMuonL2SelectorForL3IO_H
+#define RecoMuon_HLTMuonL2SelectorForL3IO_HLTMuonL2SelectorForL3IO_H
+
+/**  \class HLTMuonL2SelectorForL3IO
+ * 
+ *   L2 muon selector for L3 IO:
+ *   finds L2 muons not previous converted into L3 muons
+ *
+ *   \author  Benjamin Radburn-Smith - Purdue University
+ */
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/TrackReco/interface/TrackFwd.h"
+
+namespace edm {class ParameterSet; class Event; class EventSetup;}
```

Comment:
Hi Benjamin, yes: ok to forward declare this package entities, not ok to forward declare from other packages.
It's somewhat of a minor comment here though.